app/assets/stylesheets/_core.scss

```diff
@@ -0,0 +1,153 @@
+// Core global styles
```

Comment:
It's probably out of scope for this PR, but while much of this file is obviously needed I think we should consider switching out the base styles (`p` and headers) for components where possible. Same for some of the styles elsewhere. But probably for a followup PR.